gst/droidcamsrc/gstdroidcamsrc.c

```diff
@@ -2207,3 +2226,53 @@ gst_droidcamsrc_find_picture_resolution (GstDroidCamSrc * src,
 
   return ret;
 }
+
+gboolean
+gst_droidcamsrc_is_zsl_enabled (GstDroidCamSrc * src)
+{
+  return src->image_mode & GST_DROIDCAMSRC_IMAGE_MODE_ZSL
+      || src->image_mode & GST_DROIDCAMSRC_IMAGE_MODE_ZSL_AND_HDR;
+}
+
+gboolean
+gst_droidcamsrc_is_hdr_enabled (GstDroidCamSrc * src)
+{
+  return src->image_mode & GST_DROIDCAMSRC_IMAGE_MODE_HDR
+      || src->image_mode & GST_DROIDCAMSRC_IMAGE_MODE_ZSL_AND_HDR;
+}
+
+
+static gboolean
+gst_droidcamsrc_is_zsl_and_hdr_supported (GstDroidCamSrc * src)
```

Comment:
Not part of api offering, what does that mean? Nothing outside gst-droid uses it?